storagetransfer/conftest.py

```diff
@@ -32,30 +32,76 @@ def project_id():
     yield os.environ.get("GOOGLE_CLOUD_PROJECT")
 
 
-@pytest.fixture(scope='module')
+# cache secret from secret manager
+aws_secret_manager_cache = None
+
+
+def aws_retrieve_from_secret_manager(name: str):
+    """
+    Retrieve a secret given a name.
+
+    example ``name`` = ``projects/123/secrets/my-secret/versions/latest``
+    """
+
+    global aws_secret_manager_cache
+
+    if aws_secret_manager_cache:
+        return aws_secret_manager_cache
+
+    client = secretmanager.SecretManagerServiceClient()
+
+    # retrieve from secret manager
+    response = client.access_secret_version(request={"name": name})
+
+    # parse secret from secret manager
+    secret = json.loads(response.payload.data.decode("UTF-8"))
+
+    aws_secret_manager_cache = {
+        'aws_access_key_id': secret['AccessKeyId'],
+        'aws_secret_access_key': secret['SecretAccessKey']
```

Comment:
nit: The last item in a list of items (in `list`, `dict`, `set`, etc) should end with a comma.
```suggestion
        'aws_secret_access_key': secret['SecretAccessKey'],
```